File: app/src/main/java/com/OverSadBoy/samurairise/model/database/IDataBase.java
package com.OverSadBoy.samurairise.model.database;

import java.util.List;

public interface IDataBase {
    void addAlarms(Item item);
    Item getAlarm(int id);
    List<Item> getAllAlarms();
    int getAlarmsCount();
    int updateAlarms(Item item);
    void deleteAlarm(Item item);
    void deleteAll();
}

File: app/src/main/java/com/OverSadBoy/samurairise/model/database/IDataBase.kt
package com.OverSadBoy.samurairise.model.database

import androidx.test.runner.AndroidJUnit4
import org.junit.runner.RunWith

interface IDataBase {
    open fun addAlarms(item: Item?)
    open fun getAlarm(id: Int): Item?
    open fun getAllAlarms(): MutableList<Item?>?
    open fun getAlarmsCount(): Int
    open fun updateAlarms(item: Item?): Int
    open fun deleteAlarm(id: Int)
    open fun deleteAll()
}